### DISCUSSION TITLE
Frigate LXC - "audio_encoder_process_init: failed to get AAC encoder"

### DISCUSSION THREAD

**Discussion by raeserfisk** [tteck/Proxmox#2964] [Help Needed]
Hi!

Looking through the logs under NGINX I find the following everytime I boot up the LXC:
2024/05/11 19:04:15 [warn] 292#292: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it
2024/05/11 19:04:15 [warn] 197#197: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it
2024/05/11 19:04:15 [warn] 293#293: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it
2024/05/11 19:04:15 [warn] 160#160: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it
2024/05/11 19:04:15 [warn] 161#161: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it
2024/05/11 19:04:15 [warn] 180#180: audio_encoder_process_init: failed to get AAC encoder, audio encoding is disabled. recompile libavcodec with libfdk_aac to enable it

The error is not there if I use the same hardware but run through Docker LXC instead (but this provides different challenges so I would prefer the LXC-way).

Coincidentally I also have AAC-related issues sporadically during the day - unsure if they are related to the above. I do have sound incoming both Live and in Clips.

**Comment by SubNoizey** [comment]
Did you sort this/find a fix?

**Reply by raeserfisk** [reply]
I left the Proxmox server solution and are now using Unraid -> Docker -> Frigate.

I've given up the idea of a clean log so I am now focusing on the status symbol in the bottom right introduced in 0.14. If its green then I am happy :)
